src/components/questionary/questionaryComponents/SampleDeclaration/QuestionSubtemplateForm.tsx

```diff
@@ -64,42 +61,6 @@ export const QuestionSubtemplateForm: FormComponent<Question> = props => {
           />
 
           <TitledContainer label="Options">
-            <FormControl fullWidth margin="normal">
-              <InputLabel htmlFor="config.templateCategory">
-                Template category
-              </InputLabel>
-              <Field
-                name="config.templateCategory"
-                type="text"
-                component={Select}
-                data-cy="templateCategory"
-                inputProps={{
-                  onChange: (e: any) => {
-                    const categoryId = e.target.value;
-                    setSelectedCategory(categoryId);
-                  },
-                }}
-              >
-                {categories.map(category => {
-                  if (
-                    category.categoryId ===
-                    TemplateCategoryId.PROPOSAL_QUESTIONARY
-                  ) {
-                    return null;
-                  }
-
-                  return (
-                    <MenuItem
-                      value={category.categoryId}
-                      key={category.categoryId}
-                    >
-                      {category.name}
-                    </MenuItem>
-                  );
-                })}
-              </Field>
-            </FormControl>
-
```

Comment:
I'm not sure why this block was deleted. Did it interfere with the interval questions?

Reply:
I'll move this change do a different pull-request, as it does not really make sense to be part of "Adding interval component"